### PR TITLE
Use lazyjson payload for feedstock info

### PR DESCRIPTION
**Pull request by CJ-Wright**
This will put the feedstock data into their own files
making it easier to read/write to them via third party tools.
Note that this needs a migration of our existing graph into this structure.

### CODE REVIEW

File: conda_forge_tick/auto_tick.xsh [hunk 1]
```diff
@@ -368,6 +371,7 @@ def add_rebuild_blas(migrators, gx):
     total_graph = copy.deepcopy(gx)
 
     for node, attrs in gx.node.items():
+        attrs = attrs['payload']
```
Comment (scopatz) [inline conda_forge_tick/auto_tick.xsh]: Maybe this should be `attrs = node_attrs["payload"]` and similar cases below.

**Comment by scopatz** [comment]
Looks reasonable, but I am unsure of migration path

**Comment by CJ-Wright** [comment]
Same as the other graph migration.

**Comment by CJ-Wright** [comment]
The migration will use the https://github.com/regro/cf-scripts/pull/585/files#diff-09a07bc2e2c9570b3861d6755dff6a95 script to move everything over to the payload system.

**Comment by CJ-Wright** [comment]
Failing because upstream graph is not migrated yet.

**Pull request merged**